README.md

```diff
@@ -3,28 +3,26 @@
 [![GitHub Build Status](https://github.com/cisagov/ansible-role-debian-archive/workflows/build/badge.svg)](https://github.com/cisagov/ansible-role-debian-archive/actions)
 [![CodeQL](https://github.com/cisagov/ansible-role-debian-archive/workflows/CodeQL/badge.svg)](https://github.com/cisagov/ansible-role-debian-archive/actions/workflows/codeql-analysis.yml)
 
-This is a skeleton project that can be used to quickly get a new
-[cisagov](https://github.com/cisagov) Ansible role GitHub project
-started.  This skeleton project contains
-[licensing information](LICENSE), as well as
-[pre-commit hooks](https://pre-commit.com) and
-[GitHub Actions](https://github.com/features/actions) configurations
-appropriate for an Ansible role.
+This Ansible role adds the [Debian
+Archive](https://archive.debian.org/debian/) [package
+repositories](https://archive.debian.org/README) for supported Debian
+releases and removes the standard Debian package repositories.  It is
+intended for use with Debian releases that are no longer supported.
+
+> [!IMPORTANT] When required, this Ansible role must be applied
+> *before*
```

Comment:
[Alert](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#alerts) headers must be on their own line.

```suggestion
> [!IMPORTANT]
> When required, this Ansible role must be applied *before*
```